plugins/modules/clustering/nomad/nomad_job.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: FERREIRA Christophe (@chris93111)
+version_added: "1.2.0"
+short_description: Launch a Nomad Job
+description:
+    - Launch a Nomad job. See
+      (https://www.nomadproject.io/api-docs/jobs/) for an overview.
```

Comment:
```suggestion
    - Launch a Nomad job.
```
Please, put this in `seealso:` https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block
e.g.
```
seealso:
- name: APIC Management Information Model reference
  description: Complete reference of the APIC object model.
  link: https://developer.cisco.com/docs/apic-mim-ref/
```